### PR TITLE
FUSETOOLS-2862 - remove radio buttons for selecting templates

### PR DESCRIPTION
**Pull request by apupier**
Signed-off-by: Aurélien Pupier <apupier@redhat.com>

# Pull Request Checklist

After this checklist is all checked or PR provides explanations for possible pass-through, please put the label "Ready for review" on the PR.

## General

- [x] Did you use the Jira Issue number in the commit comments?
- [x] Did you set meaningful commit comments on each commit?
- [x] Did you sign off all commits for this PR? (git commit -s -m "jira issue number - your commit comment")

## Functional

- [ ] Did the CI job report a successful build?
- [ ] Is the non-happy path working, too?

## Maintainability

- [ ] Are all Sonar reported issues fixed or can they be ignored?
- [x] Is there no duplicated code?
- [x] Are method-/class-/variable-names meaningful?

## Tests

- [ ] Are there unit-tests?
- [ ] Are there integration tests (or at least a jira to tackle these)?
- [ ] Do we need a new UI test?

## Legal

- [x] Have you used the correct file header copyright comment?
- [x] Have you used the correct year in the headers of new files?

### CODE REVIEW

**Comment by apupier** [comment]
result of this iteration:
image
image


--> still not filtered by Runtime so I think that it makes sense to keep it grouped by Category and also keep the filter field, (and maybe it will be worthy to keep the filter field in general even if it wasn't part of the mockup)
see https://issues.jboss.org/browse/FUSETOOLS-2861 for Runtime filtering

**Comment by lhein** [comment]
looks good

**Comment by apupier** [comment]
testPR

File: editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/wizards/FuseIntegrationProjectWizard.java [hunk 1]
```diff
@@ -87,14 +88,9 @@ private NewFuseIntegrationProjectMetaData getProjectMetaData() {
 		}
 		metadata.setCamelVersion(runtimeAndCamelVersionPage.getSelectedCamelVersion());
 		metadata.setTargetRuntime(runtimeAndCamelVersionPage.getSelectedRuntime());
-		metadata.setDslType(templateSelectionPage.getDSL());
```
Comment (lhein) [inline editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/wizards/FuseIntegrationProjectWizard.java]: yay, never liked that part

File: editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/wizards/pages/FuseIntegrationProjectWizardRuntimeAndCamelPage.java [hunk 2]
```diff
@@ -172,10 +175,14 @@ public void keyReleased(KeyEvent e) {
 				setMessage(null);
 			}
 		});
-
-		camelVersionComboViewer.addSelectionChangedListener(event -> {
-			validate();
-			((FuseIntegrationProjectWizardTemplatePage)getWizard().getPage(Messages.newProjectWizardTemplatePageName)).refresh(camelVersionCombo.getText());
+		
+		camelVersionCombo.addModifyListener(new ModifyListener() {
+
```
Comment (lhein) [inline editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/wizards/pages/FuseIntegrationProjectWizardRuntimeAndCamelPage.java]: no lambda?

**Comment by lhein** [comment]
+1 on green build

**Comment by apupier** [comment]
testPR

**Comment by apupier** [comment]
image

**Comment by apupier** [comment]
> +1 on green build

**Pull request merged**